tests/FakeItEasy.Analyzer.Tests.Shared/Helpers/DiagnosticVerifier.Helper.cs

```diff
@@ -175,7 +179,7 @@ private static Project CreateProject(string[] sources, string language = Languag
 #elif VISUAL_BASIC
                     .WithProjectCompilationOptions(
                         projectId,
-                        new VisualBasicCompilationOptions(OutputKind.DynamicallyLinkedLibrary))
+                        new VisualBasicCompilationOptions(OutputKind.DynamicallyLinkedLibrary, optionStrict: OptionStrict.On))

```

Comment:
Should avoid a bit of head scratching in VB analyzer tests... For instance, with `Option Strict` off, VB was perfectly happy to silently convert `Integer` to `Byte` 😕